src/run/handlers/request-context.cts

```diff
@@ -41,7 +45,20 @@ type RequestContextAsyncLocalStorage = AsyncLocalStorage<RequestContext>
 export function createRequestContext(request?: Request, context?: FutureContext): RequestContext {
   const backgroundWorkPromises: Promise<unknown>[] = []
 
+  const isDebugRequest =
+    request?.headers.has('x-nf-debug-logging') || request?.headers.has('x-next-debug-logging')
```

Comment:
Just comparing this to https://github.com/opennextjs/opennextjs-netlify/pull/2769/files
Should this be `debug-x-nf-logging`?